lib/suggest.js

```diff
@@ -11,7 +11,7 @@ function suggest (opts) {
 
     const term = encodeURIComponent(opts.term);
     const options = Object.assign({
-      url: `https://market.android.com/suggest/SuggRequest?json=1&c=3&query=${term}`,
+      url: `https://market.android.com/suggest/SuggRequest?json=1&c=3&query=${term}&hl=${opts.lang || 'en'}&gl=${opts.country || 'us'}`,
```

Comment:
Can you do the `||` to set default options outside of the url? Check how it's done [here](https://github.com/facundoolano/google-play-scraper/blob/dev/lib/app.js#L16-L17).

Reply:
Or better, using Object.assign (see [this](https://github.com/facundoolano/google-play-scraper/issues/238))

Reply:
Don't quite see where `Object.assign` would make sense, except maybe to completely copy `opts`, but I put it outside like for `term`.